test/test_quantity_concept.py

```diff
@@ -3,20 +3,21 @@
 import unittest
 import numpy as np
```

Comment:
Rename to `test_quantity` make a pytest structure, split into named test functions.